### PR TITLE
Toy Update

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

Adds action figures of the prisoner and psychologist role
Adds missing action figures to Delta and Kilo, moves some around
Fixes Intento high score system not working well
Resprites all mech figures, repaths them, adds ones for mechs that are missing

## Why It's Good For The Game

its an epic win

## Changelog
:cl:
expansion: new mech action figures and job action figures
fix: Intento leaderboard works good
imageadd: new sprites for mech action figures
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1321,7 +1320,7 @@
 /obj/item/toy/figure/chef
 	name = "Chef action figure"
```
Comment (coiax) [inline code/game/objects/items/toys.dm:1321]:
```suggestion
	name = "\improper Chef action figure"
```

Do this for the others.

File: code/modules/holiday/easter.dm [hunk 2]
```diff
@@ -158,37 +158,26 @@
 	icon_state = "egg-[eggcolor]"
 
 /obj/item/food/egg/proc/dispensePrize(turf/where)
```
Comment (coiax) [inline code/modules/holiday/easter.dm:160]: How do you feel about just making this the arcade toy drop list, plus some additional bunny related items? Just a though, not sure if it's a good idea just for refactoring's sake.

File: code/game/objects/items/toy_mechs.dm [hunk 3]
```diff
@@ -11,14 +11,19 @@
...
 	icon = 'icons/obj/toy.dmi'
-	icon_state = "ripleytoy"
+	icon_state = "fivestarstoy"
 	verb_say = "beeps"
 	verb_ask = "beeps"
 	verb_exclaim = "beeps"
 	verb_yell = "beeps"
 	w_class = WEIGHT_CLASS_SMALL
+	/// Number of the toy, used in it's desc
+	var/series_number = 0
```
Comment (coiax) [inline code/game/objects/items/toy_mechs.dm:26]: If the series numbers actual value is unimportant, would it be possible to do something like.

(pseduocode)
```
mech_types = subtypesof(/obj/item/toy/mecha)
series_number = mech_types.IndexOf(src.type)
```

Then you wouldn't need to specify it in each type, or have the ALL_MECH_TOYS define?

Comment (Fikou) [reply code/game/objects/items/toy_mechs.dm:26]: i tried to number and order it in a specific way and i dont know if that would work with subtypesof

File: code/modules/holiday/easter.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/holiday/easter.dm:160]: i think its kind of out of scope for this, not sure

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -1267,195 +1266,205 @@
...
 
 /obj/item/toy/figure/bartender
-	name = "Bartender action figure"
+	name = "\improper Bartender action figure"
 	icon_state = "bartender"
 	toysay = "Where is Pun Pun?"
 
 /obj/item/toy/figure/borg
-	name = "Cyborg action figure"
+	name = "C\improper yborg action figure"
```
Comment (Fikou) [inline code/game/objects/items/toys.dm:1290]:
```suggestion
	name = "\improper Cyborg action figure"
```

**Comment by maxymax13** [comment]
janitor is now soulless 0/10

File: code/game/objects/items/toy_mechs.dm (hunk 3 above)
Comment (coiax) [reply code/game/objects/items/toy_mechs.dm:26]: Ah, if the numbering is important, then keep it.

Comment (tralezab) [reply code/game/objects/items/toy_mechs.dm:26]: i will pr the element that does it

**Review by Krysonism — APPROVED** [review]
looks ok!

**Review by Kylerace — APPROVED** [review]

**Pull request merged**